### PR TITLE
Update health violation tutorial

### LINKED ISSUE #24830: Update Tutorial: Classify health violations with Model Builder
- Update MBConfig
- Update screenshots
- Review other outdated information and update accordingly.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f7fff31d-9b11-e1f1-f4c1-d798a7d9b564
* Version Independent ID: 8c94de21-2cc2-fb08-314d-6a62af1f0421
* Content: [Tutorial: Classify health violations with Model Builder - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/health-violation-classification-model-builder)
* Content Source: [docs/machine-learning/tutorials/health-violation-classification-model-builder.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/tutorials/health-violation-classification-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by jwood803**
Bring the health violation Model Builder tutorial up-to-date

Fixes #24830 #22832

[Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/health-violation-classification-model-builder?branch=pr-en-us-25505)

### CODE REVIEW

**Comment by jwood803** [comment]
@luisquintanilla I believe this is ready for an initial review.

File: docs/machine-learning/tutorials/health-violation-classification-model-builder.md [hunk 1]
```diff
@@ -59,86 +60,118 @@ Each row in the dataset contains information regarding violations observed durin
...
+    ```powershell
+    $body = @{
+        InspectionType="Reinspection/Followup"
+        ViolationDescription="Inadequately cleaned or sanitized food contact surfaces"
+    }
+
+    Invoke-RestMethod "https://localhost:<PORT>/predict" -Method Post -Body ($body | ConvertTo-Json) -ContentType "application/json"
+    ```
+
+1. If successful, the output should look similar to the text below:
```
Comment (luisquintanilla) [inline docs/machine-learning/tutorials/health-violation-classification-model-builder.md:168]: I'd add some information here to explain what the output means. Especially the score.

**Review by luisquintanilla — COMMENTED** [review]
Thanks @jwood803. Looks great. Added a comment to explain the output a bit more.

File: docs/machine-learning/tutorials/health-violation-classification-model-builder.md (hunk 1 above)
Comment (jwood803) [reply docs/machine-learning/tutorials/health-violation-classification-model-builder.md]: Added some info here, but I'm not real sure if it's enough.

**Review by luisquintanilla — APPROVED** [review]
LGTM

**Pull request merged**